tests/tests_certificate.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: test - setup
+  hosts: all
+  tasks:
+    - name: Download current linux-system-roles.certificate
+      git:
+        repo: https://github.com/linux-system-roles/certificate/
+        dest: roles/linux-system-roles.certificate
+        version: master
+        depth: 1
+      delegate_to: 127.0.0.1
```

Comment:
so `localhost` didn't work, but `127.0.0.1` did???  That's very surprising - we use `localhost` delegation in many roles.

Reply:
hmm - not sure what's going on here

note that this will have to change in order to run in downstream testing - in that environment, the roles are installed from the system roles RPM, so all of the roles will be present, so you will just be able to include_role without using git